lib/BuildSystem/SwiftTools.cpp

```diff
@@ -220,6 +223,11 @@ class SwiftCompilerShellCommand : public ExternalCommand {
     if (isLibrary) {
       result.push_back("-parse-as-library");
     }
+    if (enableWholeModuleOptimization) {
+      result.push_back("-whole-module-optimization");
+      result.push_back("-num-threads");
+      result.push_back("1");
```

Comment:
Ok. I dont think its going to matter if output file has entries for individual objects when non-parallel WMO is used. Though I'll double check to be sure.